pandas/core/arrays/base.py

```diff
@@ -1207,7 +1215,7 @@ def _formatter(self, boxed: bool = False) -> Callable[[Any], str | None]:
     # Reshaping
     # ------------------------------------------------------------------------
 
-    def transpose(self, *axes) -> ExtensionArray:
+    def transpose(self, axes: NumpyAxis = None) -> ExtensionArray:
```

Comment:
Changing from `*axes` to `axes` makes it inconsistent with numpy?

Reply:
@jorisvandenbossche No, the original `*axes` argument was inconsistent with numpy

Reply:
The method `np.ndarray.transpose` seems to have it as `*axes` and not `axes`? (this is different from the `np.transpose` function)

Now, not that it matters much here, as for Series/DataFrame we disallow any specification of `axes` .. (and for the arrays we just ignore it)

Reply:
I've reverted this, since I think "compatibility with numpy" means with `ndarray.transpose`.  Now the argument is just `int`